prow/github/helpers.go

```diff
@@ -82,3 +82,25 @@ func PermissionsFromLevel(permission RepoPermissionLevel) RepoPermissions {
 		return RepoPermissions{}
 	}
 }
+
+func MatchCardToContent(cardURL, contentURL string) bool {
+	if strings.Compare(cardURL, contentURL) == 0 {
+		return true
```

Comment:
Is this valid? In what cases is this hit?

Reply:
Let me add //TO DO - if needed we can iterate this later. URL parsing is not as predictable I believe.

Reply:
I think making this work correctly is a pre-requisite to merging this pull request. sorry